tests/console/force_cron_run.pm

```diff
@@ -30,14 +30,20 @@ sub settle_load {
 
 sub run {
     select_console 'root-console';
-
     # show dmesg output in console during cron run
     assert_script_run "dmesg -n 7";
 
     # Make sure there's no load before we trigger one via cron.
     settle_load;
     my $before = time;
-    assert_script_run "bash -x /usr/lib/cron/run-crons", 1000;
+    # "bash -x /usr/lib/cron/run-crons" doesn't work really on a running system with online duration shorter than 15 minutes
+    # we stop and mask btrfs systemd timers. So keep bash -x /usr/lib/cron/run-crons for SLES 12
+	if (is_sle('<15') || is_leap('<15')) { 
+        assert_script_run("bash -x /usr/lib/cron/run-crons", 1000);
+    } 
+	else {
+		 assert_script_run('rpm -q btrfsmaintenance && systemctl start btrfs-balance.service && systemctl stop btrfs-* && systemctl mask btrfs-*');
```

Comment:
why do you stop the service immediately after starting? I think you can only stop and mask the services after L49, i.e. after `settle_load`.